### ISSUE TITLE
1.0.1-Beta01 not working

### ISSUE THREAD

**Issue by Nailik** [Gurupreet/FontAwesomeCompose#8]
It's not published on jitpack.io because there was a build error.
https://jitpack.io/com/github/Gurupreet/FontAwesomeCompose/1.0.1-beta01/build.log

**Comment by Gurupreet** [comment]
Yes it happens because of Gradle version 7 or above. Let me publish it today with latest version.
